packages/workflow-core/package.json

```diff
@@ -41,6 +41,7 @@
   "devDependencies": {
     "@babel/core": "7.17.9",
     "@babel/preset-env": "7.16.11",
+    "@ballerine/rules-engine-lib": "0.5.2",
```

Comment:
@alonp99 I think we should remove the `-lib` bit from the package.

Reply:
agree